contrib/inventory/ec2.py

```diff
@@ -338,8 +338,8 @@ def read_settings(self):
         else:
             self.all_elasticache_nodes = False
 
-        # boto configuration profile (prefer CLI argument)
-        self.boto_profile = self.args.boto_profile
+        # boto configuration profile (prefer environment variables then CLI argument)
+        self.boto_profile = os.environ.get('AWS_PROFILE', self.args.boto_profile)
```

Comment:
Will's new hit song, "99 problems and I *thought* I committed a fix to some of them". Thanks!